docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -37,21 +37,50 @@ To use Iceberg, you need:
 
 * Network access from the Trino coordinator and workers to the distributed
   object storage.
-* Access to a Hive metastore service (HMS).
+* Access to a Hive metastore service (HMS) or AWS Glue.
 * Network access from the Trino coordinator to the HMS. Hive
   metastore access with the Thrift protocol defaults to using port 9083.
 
 Configuration
 -------------
 
-Iceberg supports the same metastore configuration properties as the Hive connector.
-At a minimum, ``hive.metastore.uri`` must be configured:
+The connector supports two Iceberg catalog types, you may use either a Hive
```

Comment:
We never talk about "catalog types" anywhere.  Just say that two different metastores are supported.